app/presenters/user_presenter.rb

```diff
@@ -46,4 +46,8 @@ def unread_messages_count
   def admin
     user.admin?
   end
+
+  def moderator
```

Comment:
We're inside Rails, no need for forwardable, http://api.rubyonrails.org/classes/Module.html#method-i-delegate